stack.yaml.lock

```diff
@@ -6,7 +6,7 @@
 packages: []
 snapshots:
 - completed:
-    size: 524789
-    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/14/8.yaml
-    sha256: 8af5eb80734f02621d37e82cc0cde614af2ddc9c320610acb0b1b6d9ac162930
-  original: lts-14.8
+    size: 563098
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/17/1.yaml
+    sha256: 395775c03e66a4286f134d50346b0b6f1432131cf542886252984b4cfa5fef69
+  original: lts-17.1
```

Comment:
`stack.yaml.lock` is not part of the other repos. Should I remove it from this repo and add it to `.gitignore`?

Reply:
This is actually a public fork of the `phone-numbers` library, so I would keep `stack.yaml.lock`.

You can try upstreaming this change. Be sure to base it on the upstream master then as our patch was not included yet. Not sure if it will get accepted, as it's a bit arbitrary. If it doesn't get accepted, not sure what to do with this branch. It proves that the library works with GHC 8.10.3, but we already have proof of that when the GHC upgrade in our other repos is merged.

Reply:
Right, I realised this for https://github.com/piesync/gRPC-haskell but somehow forgot that it's also the case for this repo. Maybe because of the commit in this repo that hasn't been upstreamed yet.

Anyway, I might try to upstream the change. But yeah, this branch doesn't have to be merged, I'll close it.